ipn/ipnlocal/local.go

```diff
@@ -1220,6 +1229,10 @@ func (b *LocalBackend) SetControlClientStatus(c controlclient.Client, st control
 		prefs.WantRunning = true
 		prefs.LoggedOut = false
 	}
+	if shouldSuggest {
```

Comment:
I think `if shouldAutoExitNode() && b.lastSuggestedExitNode == ""` should be more robust